dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -24778,1696 +24779,2928 @@ BEGIN
+    65441 "Litera hangul de jumătate de lățime kiyeok"
+    65442 "Litera hangul de jumătate de lățime ssangkiyeok"
+    65443 "Litera hangul de jumătate de lățime kiyeok-sios"
+    65444 "Litera hangul de jumătate de lățime nieun"
+    65445 "Litera hangul de jumătate de lățime nieun-cieuc"
+    65446 "Litera hangul de jumătate de lățime nieun-hieuh"
+    65447 "Litera hangul de jumătate de lățime tikeut"
+    65448 "Litera hangul de jumătate de lățime ssangtikeut"
+    65449 "Litera hangul de jumătate de lățime rieul"
+    65450 "Litera hangul de jumătate de lățime rieul-kiyeok"
+    65451 "Litera hangul de jumătate de lățime rieul-mieum"
+    65452 "Litera hangul de jumătate de lățime rieul-pieup"
+    65453 "Litera hangul de jumătate de lățime rieul-sios"
+    65454 "Litera hangul de jumătate de lățime rieul-thieuth"
+    65455 "Litera hangul de jumătate de lățime rieul-phieuph"
+    65456 "Litera hangul de jumătate de lățime rieul-hieuh"
+    65457 "Litera hangul de jumătate de lățime mieum"
+    65458 "Litera hangul de jumătate de lățime pieup"
+    65459 "Litera hangul de jumătate de lățime ssangpieup"
+    65460 "Litera hangul de jumătate de lățime pieup-sios"
+    65461 "Litera hangul de jumătate de lățime sios"
+    65462 "Litera hangul de jumătate de lățime ssangsios"
+    65463 "Litera hangul de jumătate de lățime ieung"
+    65464 "Litera hangul de jumătate de lățime cieuc"
+    65465 "Litera hangul de jumătate de lățime ssangcieuc"
+    65466 "Litera hangul de jumătate de lățime chieuch"
+    65467 "Litera hangul de jumătate de lățime khieukh"
+    65468 "Litera hangul de jumătate de lățime thieuth"
+    65469 "Litera hangul de jumătate de lățime phieuph"
+    65470 "Litera hangul de jumătate de lățime hieuh"
+    65474 "Litera hangul de jumătate de lățime a"
+    65475 "Litera hangul de jumătate de lățime ae"
+    65476 "Litera hangul de jumătate de lățime ya"
+    65477 "Litera hangul de jumătate de lățime yae"
+    65478 "Litera hangul de jumătate de lățime eo"
+    65479 "Litera hangul de jumătate de lățime e"
+    65482 "Litera hangul de jumătate de lățime yeo"
+    65483 "Litera hangul de jumătate de lățime ye"
+    65484 "Litera hangul de jumătate de lățime o"
+    65485 "Litera hangul de jumătate de lățime wa"
+    65486 "Litera hangul de jumătate de lățime wae"
+    65487 "Litera hangul de jumătate de lățime oe"
+    65490 "Litera hangul de jumătate de lățime yo"
+    65491 "Litera hangul de jumătate de lățime u"
+    65492 "Litera hangul de jumătate de lățime weo"
+    65493 "Litera hangul de jumătate de lățime we"
+    65494 "Litera hangul de jumătate de lățime wi"
+    65495 "Litera hangul de jumătate de lățime yu"
+    65498 "Litera hangul de jumătate de lățime eu"
+    65499 "Litera hangul de jumătate de lățime yi"
+    65500 "Litera hangul de jumătate de lățime i"
+    65504 "Semn de cenți cu lățime completă"
+    65505 "Semn de liră cu lățime completă"
+    65506 "Semnul nu cu lățime completă"
```

Comment:
`Semnul nu cu lățime completă` sounds kinda robotic IMHO, for negation it is more natural if you say it `Semnul cu lățime incompletă`

Reply:
I fixed it.